package.json

```diff
@@ -5,7 +5,10 @@
   },
   "dependencies": {
     "@celo/contractkit": "^1.0.0",
+    "expo-linear-gradient": "^9.2.0",
```

Comment:
These dependencies shouldn't be in the root part of the project.  They were probably added by mistake at root then reinstalled at /client level.  Let's erase these lines so they're not added every time we run 'yarn add' at the root of the project.